org.eclipse.winery.repository.rest/src/main/java/org/eclipse/winery/repository/rest/resources/servicetemplates/topologytemplates/TopologyTemplateResource.java

```diff
@@ -174,6 +175,11 @@ public Response mergeWithOtherTopologyTemplate(String strOtherServiceTemplateQNa
         return Response.noContent().build();
     }
 
+    @Path("edmm/")
```

Comment:
I would use only `edmm` without the trailing slash.

Reply:
Fixed it for all endpoints in this resource file